oc-cluster

```diff
@@ -253,7 +303,15 @@ function up {
 
     echo "$CMDLINE" > $OPENSHIFT_PROFILES_DIR/$_profile/run
     echo "$(<$OPENSHIFT_PROFILES_DIR/$_profile/run)"
-    $CMDLINE
+    if [ ! -z $DEV_DOCKER_HOST ]; then
+      echo "Running in developer mode, using DOCKER_HOST=$DEV_DOCKER_HOST"
+      if [[ "$__PLATFORM" == "linux" ]]; then
+        CMDLINE="DOCKER_HOST=$DEV_DOCKER_HOST $CMDLINE"
+      fi
+      $CMDLINE -e DOCKER_HOST=$DEV_DOCKER_HOST
```

Comment:
On an existing profile, this results in:
DOCKER_HOST=blah . [/path/to/profile]/run -e DOCKER_HOST=blah
which is not valid in Linux.
If you want to source the run command, then you should likely just set the DOCKER_HOST at the beginning of that
OR
you can assemble a script that prepends `export DOCKER_HOST=blah` to the contents of the run command, and then source that.